Source/WebCore/platform/graphics/gstreamer/TrackPrivateBaseGStreamer.cpp

```diff
@@ -372,7 +372,7 @@ bool TrackPrivateBaseGStreamer::updateTrackIDFromTags(const GRefPtr<GstTagList>&
     if (!gst_tag_list_get_string(tags.get(), "container-specific-track-id", &trackIDString.outPtr()))
         return false;
 
-    auto trackID = WTF::parseInteger<TrackID>(StringView { std::span { trackIDString.get(), strlen(trackIDString.get()) } });
+    auto trackID = WTF::parseInteger<TrackID>(StringView { unsafeMakeSpan(trackIDString.get(), strlen(trackIDString.get())) });
```

Comment:
It would be nice to know if track identifiers will always be ASCII, and use `StringView::fromLatin1()` in that case.

Reply:
Can call `span(trackIDString.get())`